welly/curve.py

```diff
@@ -9,13 +9,16 @@
 import numpy as np
 from scipy.interpolate import interp1d
 
-import matplotlib.pyplot as plt
-import matplotlib as mpl
-from matplotlib.patches import PathPatch
 import warnings
 
-
 from . import utils
+from .plot import plot_kde_curve
+from .plot import plot_2d_curve
+from .plot import plot_curve
+from .quality import quality_curve
```

Comment:
Imports can be grouped per module

Reply:
I though separating them was the style here.. :D I'll group the imports from plot.py and quality.py modules.